### DISCUSSION TITLE
Add Release Notes Into UI

### DISCUSSION THREAD

**Discussion by mr1716** [netbox-community/netbox#14787] [Ideas]
Where it says "UTC netbox-demo (v3.7.0)", the v3.7.0 should be clickable to a URL inside of Netbox with release notes that are the same as https://github.com/netbox-community/netbox/releases/tag/v3.7.0

**Comment by abhi1693** [comment]
You can also add a widget on your dashboard to that can get RSS feeds for the releases

![image](https://github.com/netbox-community/netbox/assets/5083532/6326ee4e-b55c-4b64-8e23-42cd7123fc6e)

**Reply by mr1716** [reply]
@abhi1693 That's 1 option. And that loads an external website, the official netbox website. Having that load in Netbox natively would help without having to visit the external netbox website. But this is for Netbox to do this without the use of the dashboard

**Reply by abhi1693** [reply]
In that case, you can also view all release notes at `https://netbox-url.com/static/docs/release-notes/`

**Reply by mr1716** [reply]
Any chance that clicking on the version number in the bottom right could redirect to that URL?

**Reply by abhi1693** [reply]
You will have to raise this as an FR. I do not think if there is a way to do this unless you fork the repo and adjust the `<a>` tags yourself

**Reply by mr1716** [reply]
Thanks. Do you know which file I would add the <a> tags to?

I am unable to create a new issue so I’ll just clone and create a PR after I find the location and add the tag.
